src/modules/Page/html_client/mod_page_login.html.twig

```diff
@@ -1,6 +1,6 @@
 {% extends "layout_public.html.twig" %}
 
-{% block meta_title %}{{ 'Log in'|trans }}{% endblock %}
+{% block meta_title %}{{ guest.system_company.name }}{{ ' - Log in'|trans }}{% endblock %}
```

Comment:
It should be come

`{{ guest.system_company.name }} - {{ 'Log in'|trans }}`

Instead

Reply:
Changed it. to the style like the others